From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 3 Jan 2019 16:22:43 -0500
Subject: printers: allow driver name to be selected from details dialog

The field can be copy-and-pasted elsewhere
and this allows screen readers to read this field.

(cherry picked from commit f4d9b5be84056bddd51ef9d60c1089cb8c0867b7)
---
 panels/printers/details-dialog.ui | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panels/printers/details-dialog.ui b/panels/printers/details-dialog.ui
index 4524d14..f1a617e 100644
--- a/panels/printers/details-dialog.ui
+++ b/panels/printers/details-dialog.ui
@@ -126,6 +126,7 @@
                   <object class="GtkLabel" id="printer_model_label">
                     <property name="halign">start</property>
                     <property name="label">HP Inkjet Delux 9000</property>
+                    <property name="selectable">True</property>
                   </object>
                   <packing>
                     <property name="name">printer_model_label</property>
